pages/common/gulp.md

```diff
@@ -0,0 +1,12 @@
+# gulp
+
+> JavaScript task runner and streaming build system.
+> Tasks are defined within gulpfile.js at the project root.
```

Comment:
Weird, I didn't see it there (I looked). I think it works, yes, and is more in the DRY spirit :+1: Thanks!